set.mm

```diff
@@ -20885,25 +20913,9 @@ primitive connectives (no defined terms) on the right-hand side.
     ( wb wal wsb stdpc4 sbbi sylib ) ABEZCFKCDGACDGBCDGEKCDHABCDIJ $.
 
   ${
-    sbimd.1 $e |- F/ x ph $.
-    sbimd.2 $e |- ( ph -> ( ps -> ch ) ) $.
-    $( Deduction substituting both sides of an implication.  (Contributed by
-       Wolf Lammen, 24-Nov-2022.) $)
-    sbimd $p |- ( ph -> ( [ y / x ] ps -> [ y / x ] ch ) ) $=
-      ( weq wi wa wex wsb imim2d anim2d eximd anim12d df-sb 3imtr4g ) ADEHZBIZS
-      BJZDKZJSCIZSCJZDKZJBDELCDELATUCUBUEABCSGMAUAUDDFABCSGNOPBDEQCDEQR $.
-  $}
-
-  ${
-    sbbid.1 $e |- F/ x ph $.
-    sbbid.2 $e |- ( ph -> ( ps <-> ch ) ) $.
-    $( Deduction substituting both sides of a biconditional.  (Contributed by
-       NM, 30-Jun-1993.)  Reduce axiom usage.  (Revised by Wolf Lammen,
-       24-Nov-2022.) $)
-    sbbid $p |- ( ph -> ( [ y / x ] ps <-> [ y / x ] ch ) ) $=
-      ( wsb biimpd sbimd biimprd impbid ) ABDEHCDEHABCDEFABCGIJACBDEFABCGKJL $.
-
-    $( Obsolete proof of ~ sbbid as of 24-Nov_2022.  (Contributed by NM,
+    sbbidOLD.1 $e |- F/ x ph $.
+    sbbidOLD.2 $e |- ( ph -> ( ps <-> ch ) ) $.
+    $( Obsolete proof of ~ sbbid as of 24-Nov-2022.  (Contributed by NM,
```

Comment:
The conventions page requires "Obsolete version..." as a marker for OLD theorems.  Recently I approved a change where a redundant antecedent was removed, so "Obsolete proof.." is too narrow as a general expression.

This was my fault, and it was on my TODO list.  But since you modify the lines anyway, could you do me a favor and correct this glitch?

Reply:
I can. But I would rather modify the conventions, with "obsolete proof" when the statement is identical and "obsolete version otherwise", similar to "alternate proof"/"alternate version". The current comments need not be changed since they will be removed within a year.
To locate them for removal, I generally search the regexp "as of ..-Oct-2021" so this wouldn't change.

What do you think ?

Reply:
If you think of this phrase as a marker, a fixed one is preferable over a variety, I think.  In the end this helps automate the removal of outdated OLD theorems  -  some time in the future.  Searching for the date will succeed most often, but if the range crosses a change of a month, or even more, this method has some drawbacks.

Reply:
thanks